From: Sebastian Ott <sebott@linux.vnet.ibm.com>
Subject: s390/css: export css_sched_sch_todo
Patch-mainline: v3.7-rc1
Git-commit: 01c5e6dc119bf2112e363477e3b056f6b54f2d79
References: bnc#792703,FATE#314095

s390/css: export css_sched_sch_todo

Allow subchannel drivers build as a module to schedule subchannel work.

Signed-off-by: Sebastian Ott <sebott@linux.vnet.ibm.com>
Acked-by: John Jolly <jjolly@suse.de>
---
 drivers/s390/cio/css.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/s390/cio/css.c
+++ b/drivers/s390/cio/css.c
@@ -405,6 +405,7 @@ static int css_evaluate_known_subchannel
 	}
 	return ret;
 }
+EXPORT_SYMBOL_GPL(css_sched_sch_todo);
 
 static void css_evaluate_subchannel(struct subchannel_id schid, int slow)
 {
